docs/api/ui/index.md

```diff
@@ -1,10 +1,19 @@
 ---
 layout: docs-api
 toc: toc-api-ui.html
-title: Editor UI Widgets
-slug: "ui widgets"
+title: Editor APIs
+slug: "ui"
 ---
 
+The editor provides a number of APIs for nodes and plugins to use to help them integrate with the editor.
+
+ - [`RED.actions`](actions) - register custom actions
+ - [`RED.events`](events) - listen for events within the editor
+ - [`RED.notify`](notification) - show notifications in the editor
```

Comment:
@knolleary This document is useful. I found that there's a link error. "notification" is not valid. "notification**s**" is correct.